test/Models/CesiumSpec.ts

```diff
@@ -296,6 +296,120 @@ describeIfSupported("Cesium Model", function () {
       );
     });
   });
+
+  describe("Cesium Terrain Extra Tests", function () {
+    // declare different variables here, we need new instances to test
+    let terria2: Terria;
+    let terriaViewer2: TerriaViewer;
+    let container2: HTMLElement;
+    let cesium2: Cesium;
+
+    beforeEach(function () {
+      terria2 = new Terria({
+        baseUrl: "./"
+      });
```

Comment:
Any reason not to use `terria`, `terriaViewer` and other top level instances?

Reply:
Yes, the reason is that we need to set some incorrect config parameters and then instatiate those top level instances. To simulate having a bad `cesiumionAccessToken` etc.

I have tried going back and using the top level instances, but it is not a neat test - and I cannot get the tests to reliably fail because there is a pretty complex initialisation sequence in Terria.. I prefer to have these instances we are running the tests on sandboxed to be sure. Does that sound OK @na9da ? Otherwise, with more familiarity with the initialisation sequence of terria there might be a way to operate on those top level instances.